examples/async_trace_test.cpp

```diff
@@ -14,17 +14,24 @@
  * limitations under the License.
  */
 #include <unifex/async_trace.hpp>
-#include <unifex/awaitable_sender.hpp>
+
 #include <unifex/on.hpp>
 #include <unifex/scheduler_concepts.hpp>
-#include <unifex/sender_awaitable.hpp>
 #include <unifex/sync_wait.hpp>
-#include <unifex/task.hpp>
+
+#include <unifex/config.hpp>
+#include <unifex/just.hpp>
 #include <unifex/timed_single_thread_context.hpp>
 #include <unifex/transform.hpp>
 #include <unifex/typed_via.hpp>
 #include <unifex/when_all.hpp>
 
+#if UNIFEX_HAVE_COROUTINES
```

Comment:
My preference would be to flip the sense of the macro here. `UNIFEX_NO_COROUTINES` has the nice property that in the long run, when all compilers support coroutines, we don't have to keep defining a macro to keep things working.